@navikt/core/react/src/pagination/Pagination.tsx

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+import cl from "classnames";
+import { Back, Next } from "@navikt/ds-icons";
+import { Button } from "..";
+
+interface PaginationProps extends React.HTMLAttributes<HTMLDivElement> {
+  /**
+   * Current page
+   */
+  page: number;
+  /**
+   * Callback when current page changes
+   */
+  onPageChange: (page: number) => void;
+  /**
+   * Total number of pages
+   */
+  count: number;
+}
+
+export const getSteps = ({ current, count }) =>
+  new Array(count)
+    .fill(null)
+    .map((_, i) => i)
+    .filter((n) => {
+      if (count < 8) {
```

Comment:
Måtte bruke litt tid for å visualisere hva som foregikk her. Kanskje noen enkle kommentarer har passet bra inn her for fremtiden?

Reply:
Ja, denne koden er ikke helt enkel å lese, men jeg tenker testene kan fungere som dokumentasjon for fremtiden.